torch/csrc/jit/passes/remove_mutation.cpp

```diff
@@ -68,11 +69,27 @@ Node* MutationRemover::createSpecialMappedOp(Node* n) {
   return new_node;
 }
 
+bool removableSetItem(Node* n) {
+  if (n->kind() != aten::_set_item ||
+      n->input(1)->node()->kind() != prim::Constant) {
+    return false;
+  }
+  if (n->inputs().at(0)->node()->kind() != prim::ListConstruct) {
+    return false;
+  }
+  int64_t index = *constant_as<int64_t>(n->input(1));
+  if (index < 0) {
```

Comment:
nitpick: do we have `normalizeIndex` somewhere?

Reply:
let me check..